services/web/server/src/simcore_service_webserver/computation_subscribe.py

```diff
@@ -96,6 +98,21 @@ async def instrumentation_message_parser(app: web.Application, data: bytes) -> N
         )
 
 
+async def events_message_parser(app: web.Application, data: bytes) -> None:
+    rabbit_message = EventRabbitMessage.parse_raw(data)
+
+    socket_messages: List[SocketMessageDict] = [
+        {
+            "event_type": SOCKET_IO_EVENT,
+            "data": {
+                "action": rabbit_message.action,
+                "node_id": f"{rabbit_message.node_id}",
+            },
+        }
+    ]
+    await send_messages(app, f"{rabbit_message.user_id}", socket_messages)
```

Comment:
so the front-end can detect when then services are restarted via the sockets? is this used in the front-end so far (e.g. indicated in the GUI)?

Reply:
Yes, the frontend receives the message and will react accordingly to it. For now a popup is displayed for a short time and then it goes away.